docs/int/quickstart/advanced/quickstart-builder-api.md

```diff
@@ -8,7 +8,7 @@ description: Run a distributed validator cluster with the builder API (MEV-Boost
 :::caution
 Charon is in an alpha state and should be used with caution according to its [Terms of Use](https://obol.tech/terms.pdf).
 
-Charon's integration with MEV-Boost is also in an alpha state and requires a non-trivial amount of configuration to get working successfully. In future this process aims to be much more automated and seamless from a user's perspective.
+Charon's integration with MEV-Boost is also in an alpha state and requires a non-trivial amount of configuration to get working successfully. In the future, this process aims to be much more automated and seamless from a user's perspective.
 :::
 
 This quickstart guide focuses on configuring the builder API for a validator and assumes you already [have a cluster up and running](docs/int/quickstart/group/index.md).
```

Comment:
This link is broken

Reply:
And that's why relative URIs are better than full path URIs :)

Reply:
Removed the link altogether